### PR TITLE
fix #804 fix loginScript form

### LINKED ISSUE #804: Login script did not show after edit

### PR DESCRIPTION
**Pull request by gpsbird**

### CODE REVIEW

**Comment by zxdong262** [comment]
Try pull the master branch first, make the file changes only show your modification.

File: src/client/components/bookmark-form/index.jsx [hunk 1]
```diff
@@ -604,7 +604,7 @@ export class BookmarkForm extends React.PureComponent {
             initialValue: loginScript
           })(
             <div>
-              <Input.TextArea rows={1} />
+                <Input.TextArea rows={1} >{loginScript}</Input.TextArea>
```
Comment (zxdong262) [inline src/client/components/bookmark-form/index.jsx]: It do not works like this, make sure you test it before submit

**Comment by zxdong262** [comment]
@gpsbird 
How do you know this would work?
And I tried your code, it works. I do googled, and read the whole antd form/input area doc, did not find anything about this.

**Comment by zxdong262** [comment]
https://github.com/electerm/electerm/issues/804

**Comment by gpsbird** [comment]
> @gpsbird
> How do you know this would work?
> And I tried your code, it works. I do googled, and read the whole antd form/input area doc, did not find anything about this.

hh, i'm new in frontend developing.i just googled and it works, so i pushed to you! i'm happy to hear it's helpful. by the way, electerm is amazing, it's easy-to-use and better than many tools

**Comment by zxdong262** [comment]
@gpsbird Thank you. Released a new version with your fix.
https://github.com/electerm/electerm/pull/808
https://github.com/electerm/electerm/releases/tag/v0.24.39

**Pull request closed without merging**